other/drawer/src/util.ts

```diff
@@ -0,0 +1,15 @@
+import { BCS } from "aptos";
+
+export const serializeVectorU64 = (arr: Array<number>) => {
+  const serializer = new BCS.Serializer();
+  serializer.serializeU32AsUleb128(arr.length);
+  arr.forEach((arg) => serializer.serializeU64(arg));
+  return serializer.getBytes();
+};
+
+export const serializeVectorU8 = (arr: Array<number>) => {
+  const serializer = new BCS.Serializer();
+  serializer.serializeU32AsUleb128(arr.length);
+  arr.forEach((arg) => serializer.serializeU8(arg));
+  return serializer.getBytes();
+};
```

Comment:
is there no helper for this? Feels like very low level

Reply:
` BCS.serializeVectorWithFunc(params.propertyKeys, 'serializeStr'),`

Reply:
yeah `serializeVectorWithFunc` doesn't work cause it doesn't support BCSSerialize... Matt wrote me these helpers